openlibrary/templates/type/edition/view.html

```diff
@@ -255,7 +255,7 @@
                     $isbns.append(value['value'])
               $if isbns:
                 <div class="edition-omniline-item">
-                  <div>ISBNs</div>
+                  <div id="title_item">ISBNs</div>
```

Comment:
IDs should appear exactly once on a page, they are unique identifiers. `class` should be used instead of `id` when applied to multiple elements.